pages/dkp/kommander/2.3/clusters/custom-domain-certificate/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: layout.pug
+navigationTitle: Custom domain and certificate
+title: Configure a custom domain and certificate
+menuWeight: 30
+excerpt: Configure a custom domain and certificate for your Management or any Managed/Attached cluster
+beta: false
+enterprise: false
+---
+
+DKP supports configuring a custom domain name for accessing the DKP UI and other platform services. Additionally, you can provide a custom certificate for each domain, or one can be issued automatically by Let's Encrypt, or other certificate authorities supporting the ACME protocol.
```

Comment:
`you can provide a custom certificate for each domain`

There is only 1 domain per cluster. So we either need to change this to `for a domain` or explain that custom domain can be configured per cluster.

Reply:
```suggestion
DKP supports configuring a custom domain name per cluster for accessing the DKP UI and other platform services. Additionally, you can provide a custom certificate for each domain, or one can be issued automatically by Let's Encrypt, or other certificate authorities supporting the ACME protocol.
```